views/budget_list.pug

```diff
@@ -0,0 +1,15 @@
+extends layout
+
+block content
+  h1= title
+  
+  ul
+  each budget in budget_list
+    - var total = budget.income - budget.expenses
+    li 
+      a(href=budget.url) #{budget.name} 
+      p Total Funds Available: #{total}
+      
+
+  else
+    li There are no budgets.
```

Comment:
Can you maybe check your text editor settings to see if there's a way to make sure that there's a newline at end of file?

Reply:
Yea I'll take a look